.github/workflows/build.yml

```
@@ -16,6 +16,10 @@ env:
     INFLUXDB_URL: http://localhost:8086
     INFLUXDB_TOKEN: admin:admin
     INFLUXDB_BUCKET: energy/autogen
+    HEROKU_STABLE_APP_API: drec-origin-api
+    HEROKU_STABLE_APP_UI: drec-origin-ui
```

As discussed in DMs, please use the variable from secrets.

```suggestion
    HEROKU_STABLE_APP_API: ${{secrets.HEROKU_STABLE_APP_API}}
    HEROKU_STABLE_APP_UI: ${{secrets.HEROKU_STABLE_APP_API}}
```